packages/documentation-site/patternfly-docs/content/get-started/release-highlights.md

```diff
@@ -6,6 +6,149 @@ section: get-started
+
+- Table
+    - **Requires opt-in.**
+    - **Example:** [Expandable table](/components/table#expandable), which has opted in.
+    - **Note:** This release does not include expansion animations for tree tables.
+
+#### Feedback 
+
+The following components contain animations that give users dynamic feedback, helping to communicate that an action has taken or will take place.  
+
+1.  Buttons
+    - **Animation:** When a button is clicked, the active state color ripples outward from the center of the button.
+    - **Example:** All buttons in our components, including in our [button examples and demos](/components/button).
+
+1.  Form validation
+    - **Animation:** When an error occurs in a form field with validation, the component "jiggles" from side to side as danger styles are applied. 
+    - **Example:** The [validated form select](/components/forms/form-select#validated), [invalid text area](/components/forms/text-area#invalid), and [invalid text input](/components/forms/text-input#invalid) examples.
+
+1.  Menu toggle validation
+    - **Animation:** When an error occurs in a menu toggle with validation, the component "jiggles" from side to side as danger styles are applied. 
+    - **Example:** The [danger option in the select with validation](/components/menus/select#with-validation) example.
+
+1.  Progress
+    - **Animation:** When an error occurs in a progress bar, the component "jiggles" from side to side as danger styles are applied. 
+    - **Example:** The "Danger variant" option in our [progress helper text example](/components/progress#helper-text).
+
+1.  Tabs 
+    - **Requires opt-in for HTML-only implementations.**
+    - **Animation:** The bottom "active tab" line slides between tabs as they are selected.
+    - **Example:** All [default tabs](/components/tabs) in our examples and demos. This animation does not apply to boxed tabs.
+    - **Note:** To avoid a breaking change, this animation is opt-in for HTML-only implementations. To support this, view our [HTML guidance for animating the tab accent mark](/components/tabs#animate-current-tab-accent).
+
+##### Icons
+
+1.  Favorite
+    - **Animation:** When a favorite button is clicked, the star "pulses" and fills with color.
+    - **Example:** All favorite buttons in our examples and demos, including our [animated favorite button example](/components/button#favorite).
+
+1.  Hamburger menu 
+    - **Animation:** The middle bar of the hamburger icon transforms into an arrow to indicate whether the menu will expand or collapse when clicked. The behavior differs slightly for desktop and mobile views:
+        - **Desktop:** Both arrows appears on hover or keyboard focus as appropriate.
+        - **Mobile:** When the menu is expanded, a "collapse" arrow is displayed. When collapsed, an "expand" arrow appears on hover or focus.
+    - **Example:** All hamburger menu buttons in our examples and demos, including [our page examples](/components/page#vertical-navigation).
```

Comment:
Would we want to include the verbiage from the hamburger button example that this animation will occur automatically when in a `<Masthead>`, but otherwise the consumer needs to hook up logic with the `hamburgerVariant` property?

Also would we want to link to the actual hamburger button example as well?